src/modules/map/injection/index.js

```diff
@@ -11,13 +11,13 @@ import { OlFeatureInfoHandler } from '../components/olMap/handler/featureInfo/Ol
 
 export const mapModule = ($injector) => {
 	$injector
+		.registerSingleton('StyleService', StyleService)
```

Comment:
Singleton must be registered with an instance -> `.registerSingleton('StyleService', new StyleService())`